data/en/test_checkbox.json

```diff
@@ -165,4 +165,4 @@
             "title": ""
         }]
     }]
-}
\ No newline at end of file
+}
```

Comment:
Could you remove the new line at the end of the file? It's the default behaviour of js-beautify so when we run `yarn format` it'll remove the new line.